src/performance/dashboard/src/components/actions/ActionRunLoad.jsx

```diff
@@ -102,6 +102,14 @@ class ActionRunLoad extends React.Component {
                         }, 2000);
                         break;
                     }
+                    case 'failed': {
+                        this.setState({ timeRemaining:0, showModalRunTest: false, loadRunStatus: data, inlineTextLabel: 'Failed...' });
+                        setTimeout(() => {
+                            this.setState({ loadRunStatus: { status: 'idle' } })
+                            this.props.dispatch(addNotification({kind: KIND_ERROR, title:'Load run failed', subtitle:'An unexpected error occurred' }));
+                        }, 2000);
```

Comment:
This timeout is supposed to allow the action button time to display the inlineTextLabel text for two seconds (giving the user chance to read it) before reverting the button back to idle.

However as is,  the button will immediately revert to idle inviting the user to start a new run,  then two seconds later the animated notification will slide in which might be a bit disjointed. I think you may want to add "failed" here:

https://github.com/eclipse/codewind/blob/60a80476d918d8ea7c15098551130f38affac89d/src/performance/dashboard/src/components/actions/ActionRunLoad.jsx#L219

so that the button will render the temporary  "Failed..." text, then show the notification making better use of that timeout.